store/postgres/src/block_range.rs

```diff
@@ -135,33 +135,37 @@ impl<'a> QueryFragment<Pg> for BlockRangeUpperBoundClause<'a> {
 /// Helper for generating SQL fragments for selecting entities in a specific block range
 #[derive(Debug, Clone, Copy)]
 pub enum EntityBlockRange {
-    Mutable(BlockRange), // TODO: check if this is a proper type here (maybe Range<BlockNumber>?)
+    Mutable((BlockRange, bool)),
```

Comment:
There should be a comment explaining what the `bool` is for

Actually, since this is to indicate whether we want to compare with the upper or lower of the block range, it might be worth having an enum `{ Lower, Upper }` for this

Reply:
Yeah. Enum seems reasonable here.